doc/devel/contribute.rst

```diff
@@ -279,6 +253,63 @@ though not necessarily all at the same time:
 - It involves Python features such as decorators and context managers, which
   have subtleties due to our implementation decisions.
 
+.. _first_contribution:
+
+First contributions
+-------------------
+
+If this is your first open source contribution, or your first time contributing to Matplotlib,
+and you need help or guidance finding a good first issue, look no further. This section will
+guide you through each step.
```

Comment:
Thoughts on changing it to this instead? Colon makes more sense when structured like this in my opinion. I think you changed the period to colon because that sentence reads a bit awkward. 

If this is your first open source contribution, or your first time contributing to Matplotlib,
and you need help or guidance finding a good first issue, look no further: this section will
guide you through each step.